bin/shellmock

```diff
@@ -279,7 +279,15 @@ shellmock_replay()
    # Otherwise return the output
    #----------------------------
    else
-       $ECHO "$output" | $AWK 'BEGIN{FS="%%"}{ for (i=1;i<=NF;i++) {print $i}}'
+       local out
+       out=$($ECHO "$output" | $AWK 'BEGIN{FS="%%"}{ for (i=1;i<=NF;i++) {print $i}}')
+
+       if [ "$status" -ge 0 ] ; then
```

Comment:
Another thought here may be to allow --error to be specified separately allowing the user to have more control over what got written to the various file handles.

Reply:
Sorry, but not understood. You can explain better?

Reply:
I think I understand, you wrote. But allow --error, in my opinion, allow more mistakes. Because the user can be configured - status 0 and --error at the same time. Does not make sense. My opinion is --output is the best approach